package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "radiance-ui",
-  "version": "2.1.1",
+  "version": "3.0.0",
```

Comment:
I believe when you publish, this number gets updated to whatever version you give the newly published version. If you update this before publishing, I believe npm will think this is the current version that you're bumping up.

Once you finish the publish, it creates a commit with the updated version number and you should push that up to master directly after.